doc/api/Loading_a_Content.md

```diff
@@ -706,6 +706,44 @@ Those are the possible values for that option:
   More information about the `"RELOADING"` state can be found in
   [the player states documentation](./Player_States.md).
 
+### onAudioTrackNotPlayable
+
+_type_: `string|undefined`
+
+_defaults_: `"continue"`
+
+Specifies the behavior when all audio tracks are not playable.
```

Comment:
We could clarify by giving some example cases below

Reply:
What kind of example could be relevant here? Doesn't the description of the option provides clear indication on what the player behavior is?

> - `"continue"`: The player will proceed to play the content without audio.
> 
> - `"error"`: The player will throw an error to indicate that the audio tracks could not be
>   played.

Or maybe you want me to add an example that describes how all tracks can be unplayable (codec not supported, license not containing the keys for audio/video, ...)

Reply:
> Or maybe you want me to add an example that describes how all tracks can be unplayable (codec not supported, license not containing the keys for audio/video, ...)

Yes, this